chore(docs): update Payzen docs for M1 (#869)
front-commerce/developers.front-commerce.com: docs/03-extensions/payzen/how-to/front-commerce.mdx

After restarting Front-Commerce, you should be able to see a new payment method
called "credit card" in your checkout step.

## Specific configuration for Magento1

> Using PayZen with Magento1 require another modification on how the Payment flow
> is handled.

Reviewer: ```suggestion
When using PayZen with Magento1 it requires the `directOrder` payment flow.
```
